web-app/js/angular/directives/publication.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by Temi Varghese on 15/07/15.
+ */
+profileEditor.directive('publication', function ($browser) {
+    return {
+        restrict: 'AE',
+        require: [],
+        scope: {
+            authors: '=',
```

Comment:
You can pass an object reference to the directive so you don't need to specify individual properties.